packages/bsky/src/context.ts

```diff
@@ -13,6 +14,7 @@ export class AppContext {
   constructor(
     private opts: {
       db: Database
+      dbPrimary?: Database & Primary
```

Comment:
is there a reason this is optional? looks like it should always be set based on the service entry file?

Reply:
That was leftover from my first pass, I think I must have tidied this up while you were reviewing!

Reply:
ah nice 👍